index.js

```diff
@@ -3,11 +3,22 @@ const { DynamoDBDocument } = require('@aws-sdk/lib-dynamodb');
 
 const getItem = require('./dynamodb/get-item');
 
-const client = new DynamoDB({});
-const documentClient = DynamoDBDocument.from(client);
+function createDynamoClient(documentClient) {
+  return {
+    getItem: (tableName, key, options) =>
+      getItem(documentClient, tableName, key, options)
+  };
+}
+
+function init(dynamoDbClientConfig, translateConfig) {
```

Comment:
Should maybe default arguments to null to make it clear they are optional?

Reply:
Yes, but maybe default to undefined (like it does if you don't default it) and not null?

Reply:
Agree. It doesn't really have any impact as it would still be undefined if omitted but guess it's still useful for clarity.

An amusing side effect is that if you would pass in null it would still end up as undefined ;-)

But yes, will add default value undefined.

Reply:
Wow that is interesting, did not think/know that 🧠

Reply:
Hm, I was actually wrong, had to test it ;-) If default value is e.g. 1 and you pass undefined explicitly, the parameter value will still be 1. For all other cases incl. null it will be set to the passed value.